### ISSUE TITLE
Questions: on CTC model training

### ISSUE THREAD

**Issue by cri5Castro** [facebookresearch/wav2letter#557]
Hi there I have prepared my data for AM and LM   thinking on using ASG loss (CGLU) on character fashion so my LM is trained using my data transformed into the asg format ( 1,2 for char repetitions)
I want to try a TDS model + CTC, do you think is suitable to  just  train the AM with the same data (in asg format) or should I retrain the LM with raw. character data and  the AM with this raw character data.

Also I wanted to finetune the TDS+CTC from 2019 SOTA since my dataset is significatly small in comparison  (185 hrs ,120,500 samples for training)  it will be nice to have some hyperparams recomendations , this is my proposed setup

``
##Hyperparams
--replabel=2 \
"--surround=|" \
--iter=25 \
--batchsize=4 \
--lr=0.5 \
--momentum=0.6 \
--maxgradnorm=1 \
--onorm=target \
--sqnorm=true \
--mfsc=true \
--nthread=8 \
--criterion=ctc \
--gamma=0.5 \
--stepsize=200 \
--seed=2 \
--iter=25 \
--enable_distributed=true \
--lrcrit=0.006 \
--labelsmooth=0.05

**Comment by vineelpratap** [comment]
> my LM is trained using my data transformed into the asg format ( 1,2 for char repetitions)
> I want to try a TDS model + CTC, do you think is suitable to just train the AM with the same data (in ASG format) or should I retrain the LM with raw. character data and the AM with this raw character data.

LM should be always trained on original words. It shouldn't be dependent on the criterion. 

> I wanted to fine-tune the TDS+CTC from 2019 SOTA 

There is no need to use `--replabel=2` for CTC. CTC can handle repetition tokens. I do not see any other issues. 


Also, it would help if you could let us know these questions -
Is your new dataset also in English ? Also, are you planning to use the same word piece token set as LibriSpeech or letters ?
